core/address.go

```diff
@@ -37,6 +41,10 @@ func IsMetachainIdentifier(identifier []byte) bool {
 
 // IsSmartContractOnMetachain verifies if an address is smart contract on metachain
 func IsSmartContractOnMetachain(identifier []byte, rcvAddress []byte) bool {
+	if len(rcvAddress) <= hooks.NumInitCharactersForScAddress+numInitCharactersForOnMetachainSC {
```

Comment:
`numInitCharactersForOnMetachainSC` - I can't really decode what it means: `ForOnMetachain`, is that correct?